forcedotcom/salesforcedx-vscode-automation-tests
# Fixes for Templates on remote

Makes sure templates.e2e.ts successfully passes on remote GHA

How to test:
Go to the [End to End Tests Workflow](https://github.com/forcedotcom/salesforcedx-vscode/actions/workflows/e2e.yml) in salesforcedx-vscode repo
Paste the PR's branch (cristi/push-pull-on-remote) in the second field of the Run workflow dropdown (Set the branch to use for automation tests)
Choose templates.e2e.ts test from the dropdown to select the test to run.
AC:
Test suite should pass.

Successful runs:
- 
-
-
-
@W-13744004@
